### ISSUE TITLE
Programmatically hiding messages broken

### ISSUE THREAD

**Issue by ttill** [adopted-ember-addons/ember-notify#138]
In the past it was possible to manually remove open messages again (as documented in the readme) by doing `message.set('visible', false);`. This was broken in 436a3e526c58f1ec605169a91f59f03deaed9b33.

It would be great to have this feature back.

**Comment by btecu** [comment]
Can you do something like `message.selfClose()`?

**Comment by jameswmcnab** [comment]
Just came up against this issue, is this addon still maintained?

**Comment by btecu** [comment]
@jameswmcnab yes, I haven't had tried it yet but does `message.selfClose()` not work?

**Comment by ttill** [comment]
We do not have direct access to `message`. See #139.

**Comment by btecu** [comment]
@ttill While I don't think that's the cleanest approach, it would solve your need and @jameswmcnab. Can you get the tests to pass?

**Comment by ttill** [comment]
@btecu How would you propose to expose the close functionality?

By rebasing on master I was able to break the tests even more. Previously it only failed for some ember versions, now it fails for all of them. I do not understand how the errors would be related to my changes. Do you have any idea? Locally the tests pass. Thanks.

**Comment by btecu** [comment]
@ttill I think we need to:

Update ember-cli to latest version
Fix lint & deprecations
I took a stab at this in #152 and I got it updated and fixed a few deprecations, but there are several left. Do you have some cycles to help?

**Comment by ttill** [comment]
@btecu I will try to have it a  shot later today
